data/faqs.md

```diff
@@ -1,3 +1,14 @@
+<style>
+body {
+  -webkit-overflow-scrolling: touch;
```

Comment:
`faqs.json` is the generated version of `faqs.md`. It contains the processed text from faqs.md.